### ISSUE TITLE
start() fires on wrong elements

### ISSUE THREAD

**Issue by FoHoOV** [TarekRaafat/autoComplete.js#230]
after the new API in version 10.0.4 I tried to achieve the same old problem on comment #83 and since trigger doesn't work the same way my solution was:
```
                            trigger: () => {
                                return true;
                            },
                            events: {
                                input: {
                                    focus: (event) => {
                                        _auto.start();
                                    },
                                },
                            }
```
_auto is the object instantiated via calling `new autoComplete` and is local to every input I'm setting it to (its inside forEach).

the problem is that I have multiple inputs that require the great lib that you have created(thank you!), and when I focus on any of them the first input's 'autoComplete_list_N' gets opened not the one bound to it.

if I log the event object after clicking on  only ONE input I get the following output:

```
FocusEvent {isTrusted: true, relatedTarget: null, view: Window, detail: 0, sourceCapabilities: null, …}
FocusEvent {isTrusted: true, relatedTarget: input#code-name, view: Window, detail: 0, sourceCapabilities: InputDeviceCapabilities, …}
```

which is weird because only one `FocusEvent `should be fired.

btw you forgot to close a tag(strong) on your demo at [playground](https://tarekraafat.github.io/autoComplete.js/#/playground) on `strong `tag at `element `section ->

`info.innerHTML = `Found <strong>${data.matches.length}</strong> matching results for <strong>"${data.query}"</strong`;`

**Comment by TarekRaafat** [comment]
Hello @FoHoOV,

Thanks a lot for reporting these issues.

I'll check the event issue today and get back to you ASAP.

Cheers, and have a good day! :)

**Comment by TarekRaafat** [comment]
The issue has been resolved in [v10.1.1](https://github.com/TarekRaafat/autoComplete.js/releases/tag/v10.1.1), please check it and let me know how it goes.

**Comment by FoHoOV** [comment]
@TarekRaafat thank you the issue is resolved. thank you for your great work!
